docs/source/serialization.rst

```diff
@@ -427,6 +427,91 @@ Using a traced model for inference
 
 Using the traced model for inference is as simple as using its ``__call__`` dunder method:
 
+.. code-block:: python                                              
+
+    traced_model(tokens_tensor, segments_tensors)                   
+
+Deploying HuggingFace TorchScript models on AWS using the Neuron SDK
+-----------------------------------------------------------------------------------------------------------------------
+
+AWS introduced the `Amazon EC2 Inf1 <https://aws.amazon.com/ec2/instance-types/inf1/>`_ instance family for low cost,
+high performance machine learning inference in the cloud. The Inf1 instances are powered by the AWS Inferentia chip, a
+custom-built hardware accelerator, specializing in deep learning inferencing workloads. `AWS Neuron
+<https://awsdocs-neuron.readthedocs-hosted.com/en/latest/#>`_ is the SDK for Inferentia that supports tracing and
+optimizing transformers models for deployment on Inf1. The Neuron SDK provides:
+
+
+1. Easy-to-use API with one line of code change to trace and optimize a TorchScript model for inference in the cloud.
+2. Out of the box performance optimizations for `improved cost-performance
+   <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-guide/benchmark/>`_.
+3. Support for HuggingFace transformers models built with either `PyTorch
+   <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/src/examples/pytorch/bert_tutorial/tutorial_pretrained_bert.html>`_
+   or `TensorFlow
+   <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/src/examples/tensorflow/huggingface_bert/huggingface_bert.html>`_.
+
+
+Implications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Transformers Models based on the `BERT (Bidirectional Encoder Representations from Transformers)
+<https://huggingface.co/bert-base-uncased>`_ architecture, or its variants such as `distilBERT
```

Comment:
Can you please align those documentation links to the newest documentation, e.g. `distilbert` -> https://huggingface.co/docs/transformers/master/model_doc/distilbert